src/libraries/Microsoft.Extensions.Caching.Memory/ref/Microsoft.Extensions.Caching.Memory.cs

```diff
@@ -37,6 +40,13 @@ protected virtual void Dispose(bool disposing) { }
         public Microsoft.Extensions.Caching.Memory.MemoryCacheStatistics? GetCurrentStatistics() { throw null; }
         public void Remove(object key) { }
         public bool TryGetValue(object key, out object? result) { throw null; }
+
+#if NET9_0_OR_GREATER
+        [OverloadResolutionPriority(1)]
+        public bool TryGetValue(ReadOnlySpan<char> key, out object? result) { throw null; }
```

Comment:
The approved API calls the second parameter `value`, not `result`.

The mismatch of the parameter name between `ref` and `src` is also why the build failed.

Reply:
Will fix, thanks. I caught it in one place, not the other.

Reply:
474041ddfce80ea9eb98c2937a618f70720b2e9f